app/models/legacy/loan.rb

```diff
@@ -112,7 +112,7 @@ def public_level_option_value
 
 
 
-  MIGRATION_STATUS_OPTIONS = TransientOptionSet.new(
+  MIGRATION_STATUS_OPTIONS = Legacy::TransientOptionSet.new(
```

Comment:
These are also outstanding changes on another open PR.  And I've been needing to temporarily update in my latest branch to get the latest db migrate commands to function.  Really hoping that we can start getting branches merged down to develop and avoid duplicating already completed efforts.